### PR TITLE
Support export of CollectAndDistributeFpnRpnProposalsOp

### PR DESCRIPTION
**Pull request by orionr**
* This is somewhat of an RFP, but hopefully we'll land some variation in the near future.

### CODE REVIEW

File: tools/convert_pkl_to_pb.py [hunk 1]
```diff
@@ -183,25 +213,48 @@ def convert_op_name(op):
...
-        gen_proposals_op, ext_input = convert_gen_proposals(
-            op_in, blobs,
-            rpn_min_size=float(cfg.TEST.RPN_MIN_SIZE),
-            rpn_post_nms_topN=cfg.TEST.RPN_POST_NMS_TOP_N,
-            rpn_pre_nms_topN=cfg.TEST.RPN_PRE_NMS_TOP_N,
-            rpn_nms_thres=cfg.TEST.RPN_NMS_THRESH,
-        )
-        net.external_input.extend([ext_input])
-        return [gen_proposals_op]
+    @op_filter()
```
Comment (newstzpz) [inline tools/convert_pkl_to_pb.py]: @op_filter(type='Python')?

Comment (gadcam) [reply tools/convert_pkl_to_pb.py]: @newstzpz L218 is here to replace it if I am not mistaking

Comment (orionr) [reply tools/convert_pkl_to_pb.py]: I don't really have much preference between a direct check below and a `type='Python'` check in the decorator. Any preference across the team?

**Comment by orionr** [comment]
Okay - this should be able to land when ready. I rebased to latest and ran

```
wget https://s3-us-west-2.amazonaws.com/detectron/35857197/12_2017_baselines/e2e_faster_rcnn_R-50-C4_1x.yaml.01_33_49.iAX0mXvW/output/train/coco_2014_train%3Acoco_2014_valminusminival/generalized_rcnn/model_final.pkl -O models/e2e_faster_rcnn_R-50-C4_1x_35857197.pkl

python tools/convert_pkl_to_pb.py --use_nnpack 0 --net_name detectron_e2e_faster_rcnn_R-50-C4_1x_35857197 --cfg configs/12_2017_baselines/e2e_faster_rcnn_R-50-C4_1x.yaml --out_dir models/pb_output/e2e_faster_rcnn_R-50-C4_1x/ --test_img demo/16004479832_a748d55f21_k.jpg TEST.WEIGHTS models/e2e_faster_rcnn_R-50-C4_1x_35857197.pkl
```
to confirm existing models work.

The FPN model as follows fails on test, but likely due to GenerateProposals needing some adjustment. For the record, I ran

```
wget https://s3-us-west-2.amazonaws.com/detectron/35857345/12_2017_baselines/e2e_faster_rcnn_R-50-FPN_1x.yaml.01_36_30.cUF7QR7I/output/train/coco_2014_train%3Acoco_2014_valminusminival/generalized_rcnn/model_final.pkl -O models/e2e_faster_rcnn_R-50-FPN_1x_35857345.pkl

python tools/convert_pkl_to_pb.py --use_nnpack 0 --net_name detectron_e2e_faster_rcnn_R-50-FPN_1x_35857345 --cfg configs/12_2017_baselines/e2e_faster_rcnn_R-50-FPN_1x.yaml --out_dir models/pb_output/e2e_faster_rcnn_R-50-FPN_1x/ --test_img demo/16004479832_a748d55f21_k.jpg TEST.WEIGHTS models/e2e_faster_rcnn_R-50-FPN_1x_35857345.pkl
```
cc @yinghai

**Comment by stanleychima** [comment]
okay the file need a solution it better we join hand together and fix it

**Comment by gadcam** [comment]
@rbgirshick As #372 is in I think this one could be closed.

**Pull request closed without merging**